test-tooling/src/commonMain/kotlin/it/unibo/collektive/testing/Node.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2024, Danilo Pianini, Nicolas Farabegoli, Elisa Tronetti,
+ * and all authors listed in the `build.gradle.kts` and the generated `pom.xml` file.
+ *
+ * This file is part of Collektive, and is distributed under the terms of the Apache License 2.0,
+ * as described in the LICENSE file in this project's repository's top directory.
+ */
+
+package it.unibo.collektive.testing
+
+import it.unibo.collektive.Collektive
+import it.unibo.collektive.aggregate.api.Aggregate
+import it.unibo.collektive.networking.InboundMessage
+import it.unibo.collektive.networking.Network
+import it.unibo.collektive.networking.OutboundMessage
+
+/**
+ * A network node with an associated [environment], [id], [value], and [program].
+ */
+class Node<R>(
+    val environment: Environment<R>,
```

Comment:
Can be private

Reply:
It could, but for a testing library, I prefer to keep it accessible.